src/drivers/virtio/virtqueue/packed.rs

```diff
@@ -872,17 +872,17 @@ impl DrvNotif {
 	/// Disables notifications by unsetting the LSB.
```

Comment:
The comment refers to the previous incorrect behaviour.

Reply:
Fixed. 👍